pkg/cmd/grafana/main.go

```diff
@@ -32,11 +33,26 @@ func main() {
 		Commands: []*cli.Command{
 			gcli.CLICommand(version),
 			gsrv.ServerCommand(version, commit, enterpriseCommit, buildBranch, buildstamp),
+			{
+				// The kubernetes standalone apiserver service runner
+				// The command line is actually managed by cobra
+				Name:  "apiserver",
+				Usage: "run a standalone api service (experimental)",
+				Action: func(context *cli.Context) error {
+					return nil // not actually used
+				},
+			},
 		},
 		CommandNotFound:      cmdNotFound,
 		EnableBashCompletion: true,
 	}
 
+	//
+	if len(os.Args) > 1 && os.Args[1] == "apiserver" {
+		apiserver.RunMain()
+		return
+	}
+
```

Comment:
We can clean this up by moving this code into the Action and moving the definition of the command itself into the apiserver module

Reply:
That was my first try... the problem is that the two CLI systems initialize flags differently and i think that can only happen once.

In particular, we need the "--help" command to return the k8s options, not the grafana ones.